style/ruby/sample.rb

```diff
@@ -40,7 +40,7 @@ def method_with_single_method_block
   end
 
   def method_with_oneline_combined_methods_block
-    items.map { |item| "#{item.one} #{item.two}" }
+    items.map {|item| "#{item.one} #{item.two}"}
```

Comment:
It's going to take a while for me to get used to this. It looks so cramped :(

Should we keep the extra space at the end? I sort of think of it as going with the space after the `|`, not the space after the opening `{`.